master/buildbot/process/builder.py

```diff
@@ -279,38 +279,31 @@ def getAvailableWorkers(self):
     deprecatedWorkerClassMethod(locals(), getAvailableWorkers)
 
     @defer.inlineCallbacks
-    def canStartWithWorkerForBuilder(self, workerforbuilder, buildrequests=None):
+    def canStartBuild(self, workerforbuilder, buildrequest):
```

Comment:
Builder cannot be overriden or customized unless via patching, so I guess that one is okay